backend/src/main/java/com/woowacourse/ternoko/core/application/CrewService.java

```diff
@@ -18,8 +22,16 @@
 public class CrewService {
 
     private final CrewRepository crewRepository;
+    private final MemberRepository memberRepository;
 
     public void partUpdateCrew(final Long crewId, final CrewUpdateRequest request) {
+        final Crew crew = getCrewById(crewId);
+        final String requestNickname = request.getNickname();
+
+        if (!crew.isSameNickname(requestNickname)
+                && memberRepository.existsByNickname(request.getNickname())) {
```

Comment:
c : 조건절도 좀 길어서 가독성있게 바꿔보면 좋을 것 같아요

```suggestion
        if (isDuplicatedNickname(crew, requestNickname)) {
```